src/python_testing/TC_MCORE_FS_1_3.py

```diff
@@ -32,131 +48,221 @@
 from mobly import asserts
 
 
+class Subprocess(threading.Thread):
```

Comment:
these classes seem generic. Could we place them in https://github.com/project-chip/connectedhomeip/tree/master/src/python_testing/matter_testing_infrastructure ?

Reply:
Will do as a followup: https://github.com/project-chip/connectedhomeip/issues/35348

Also see this: https://github.com/project-chip/connectedhomeip/pull/35274#discussion_r1737236695